src/osp/Satellites/SatActiveArea.cpp

```diff
@@ -43,9 +43,3 @@ using Magnum::Matrix4;
 using namespace osp::universe;
 
 const std::string SatActiveArea::smc_name = "ActiveArea";
```

Comment:
This can live in the .h file as a "static inline std::string const smc_name = "ActiveArea";

Reply:
Though do consider if it can instead be

static constexpr std::string_view smc_name = "ActiveArea"